generator/src/lib.rs

```diff
@@ -85,13 +85,30 @@ named!(cfloat<&str, f32>,
     terminated!(nom::float_s, char!('f'))
 );
 
+macro_rules! khronos_link (
+    ($name:expr) => {
+        Literal::string(&format!("<https://www.khronos.org/registry/vulkan/specs/1.1-extensions/man/html/{name}.html>", name=$name))
+    }
+);
+
+pub fn define_khronos_link_macro() -> Tokens {
+    quote! {
+        macro_rules! khronos_link (
+            ($name:expr) => {
+                Literal::string(&format!("<https://www.khronos.org/registry/vulkan/specs/1.1-extensions/man/html/{name}.html>", name=$name))
+            }
+        );
+    }
+}
```

Comment:
If this is going to be removed in the next commit, it might as well not be introduced.

Reply:
It was introduced after appveyor failed. The commit message serves as a reminder as to why it was disabled and potentially enabling it after sorting out how to get vulkan appveyor. I can squash and force push if it's a problem.

Reply:
I'm not sure what you're referring to. I'm talking about the function definition that this comment is on.

Reply:
It's been removed